### DISCUSSION TITLE
Error code 552

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2820] [Help with using Postal] [answered]
I am trying to send an email to @txt.att.net and @vtext.com but I get the 552 error code, have anyone encountered it or how can I fix it please? Any suggestions would be appreciated 
![IMG_0429](https://github.com/postalserver/postal/assets/96216385/423d257c-1996-46e2-9f03-37d942d27af7)

**Comment by willpower232** [comment, marked as answer]
that error message suggests your IP address has found its way onto some block lists, you can use something like https://mxtoolbox.com to confirm
